yarn.lock

```diff
@@ -6281,27 +6281,30 @@ node-pre-gyp@^0.6.29, node-pre-gyp@~0.6.32:
     tar "^2.2.1"
     tar-pack "^3.4.0"
 
-node-sass@3.4.2:
-  version "3.4.2"
-  resolved "https://registry.yarnpkg.com/node-sass/-/node-sass-3.4.2.tgz#ef61069927f1578ae51408ed60298449c4cdd294"
+node-sass@^3.4:
+  version "3.13.1"
```

Comment:
Looks like this is related to bourbon-neat. While the dependency for bourbon-neat specifies `node-sass "^3.4"`, node-sass v3.13.1 gets introduced during install here: `> node-sass@3.13.1 install /Users/vacokainia/Desktop/coding/vets-website/node_modules/bourbon-neat/node_modules/node-sass`. When looking at the version of node-sass that's included in the node-modules under bourbon-neat, it is indeed 3.13.1. Not sure how to reset that, as we're not installing bourbon-neat directly (I think it's installed as a dependency of something else), so running `yarn remove bourbon-neat` to reset it doesn't work.